examples/flask/test_db_rock/rockcraft.yaml

```diff
@@ -1,28 +1,16 @@
 # Copyright 2024 Canonical Ltd.
 # See LICENSE file for licensing details.
 name: test-db-flask
-summary: A flask test app
-description: OCI image for the test flask app
+summary: Default Flask application image.
+description: Default Flask application image.
 version: "0.1"
-base: ubuntu:22.04
+base: bare
 license: Apache-2.0
-platforms:
-  amd64:
+
+extensions:
+  - flask-framework
 
 parts:
-  flask-dependencies:
-    plugin: python
-    source: .
-    stage-packages:
-      - python3-venv
-    python-packages:
-      - gunicorn
-    python-requirements:
-      - requirements.txt
-  flask-app:
-    plugin: dump
-    source: .
-    organize:
-      "*": flask/app/
+  flask-framework/install-app:
     prime:
-      - flask/app
+      - -flask/app/requirements.txt
```

Comment:
Should there be a final new line? Also, given that it is present, shouldn't it list every file that needs to be copied?

Reply:
Added, thanks. Yes, they are equivalent.

Reply:
What is meant by they are equivalent? The way I understand `prime` is that only what is specifically listed here will be copied?

Reply:
Yes, here's use the prime's exclude list feature that exclude certain file and include all other files.

Reply:
ah I see, makes sense